frontend/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -369,6 +416,192 @@ function* maybeLoadIsosurface(
   return [];
 }
 
+function* markEditedCellAsDirty(): Saga<void> {
+  const volumeTracing = yield* select(state => getActiveSegmentationTracing(state));
+  if (volumeTracing != null && volumeTracing.fallbackLayer == null) {
+    const activeCellId = volumeTracing.activeCellId;
+    modifiedCells.add(activeCellId);
+  }
+}
+
+function* refreshIsosurfaces(): Saga<void> {
+  yield* put(saveNowAction());
+  // We reload all cells that got modified till the start of reloading.
+  // By that we avoid to remove cells that got annotated during reloading from the modifiedCells set.
+  const currentlyModifiedCells = new Set(modifiedCells);
+  modifiedCells.clear();
+
+  const segmentationLayer = Model.getVisibleSegmentationLayer();
+  if (!segmentationLayer) {
+    return;
+  }
+  // First create an array containing information about all loaded isosurfaces as the map is manipulated within the loop.
+  const isosurfacesMapForLayer = isosurfacesMapByLayer[segmentationLayer.name] || new Map();
```

Comment:
If the `new Map` branch is hit, it probably should be stored in `isosurfacesMapByLayer`. Maybe something like this: 

```suggestion
  isosurfacesMapByLayer[segmentationLayer.name] = isosurfacesMapByLayer[segmentationLayer.name] || new Map();
  const isosurfacesMapForLayer = isosurfacesMapByLayer[segmentationLayer.name];
```